rugby.ipynb

```diff
@@ -0,0 +1,3579 @@
+{
```

Comment:
<p>I think the definition of <code>teams</code> should be here. Moreover, some care is in order with its values. Now for example, you will see that while the first position of <code>teams</code> is England, however, according to the csv, the <code>0</code> index corresponds to Wales! :warning:</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_example_data/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>The order of the values in teams is done with respect to their wins in each match. What criteria do you suggest for team ordering? Could you please provide the team ordering of the 6 teams?</p>

Reply:
<p>I just read the output of the posterior group for the example in arviz. As per the example the team order is <code>['Wales', 'France', 'Ireland', 'Scotland', 'Italy' 'England']</code> . Is it correct?</p>

Reply:
Yes, exactly!